src/vivarium/framework/event.py

```diff
@@ -113,7 +118,7 @@ def emit(self, index: pd.Index, user_data: Optional[Dict] = None) -> Event:
         e = Event(
             index,
             user_data,
-            self.manager.clock() + self.manager.step_size(),
+            self.manager.clock() + self.manager.step_size(),  # type: ignore[operator, arg-type]
```

Comment:
Need to sort out the clock type hints here. They include Number which don't all have __add__ implemented.